backend/papers/models/author.py

```diff
@@ -0,0 +1,21 @@
+"""author.py"""
+from django.db import models
+
+from .paper import Paper
+
+
+class Author(models.Model):
+    """Author Model"""
+    first_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    email = models.EmailField()
```

Comment:
How about adding max_length to this field?

Reply:
I added to it! Thank you :)

Reply:
By the way, it seems that default max_length of the field is 254 when not specified.

Reply:
Oh, that's good. Thank you for the information. Then I will resolve this conversation.